include/eosio/abi.hpp

```diff
@@ -197,13 +197,17 @@ struct abi_type {
    struct array {
       abi_type* type;
    };
+   struct fixed_array {
+      abi_type* type;
+      size_t    size;
+   };
    struct struct_ {
       abi_type*              base = nullptr;
       std::vector<abi_field> fields;
    };
    using variant = std::vector<abi_field>;
    std::variant<builtin, const alias_def*, const struct_def*, const variant_def*, alias, optional, extension, array,
-                struct_, variant>
+                fixed_array, struct_, variant>
```

Comment:
Doesn't this have to go at the end? Otherwise existing serialized `abi_type` will think a `struct_` is a `fixed_array`.